### DISCUSSION TITLE
Alerts on Top of Dialog Modals

### DISCUSSION THREAD

**Discussion by lirc571** [saadeghi/daisyui#3145] [Q&A]
Hi all, I would like to show alerts on top of everything else, including modals.

If I understand correctly, dialogs (and popovers) are placed on the top layer and only other top layer elements can be on top of it. However, even if I implement alerts as top layer elements, newly opened modals will go on top of the existing alerts.

Is that correct? Does that mean I have to avoid using dialogs for modals?

Anyone else having similar frustrations?

**Comment by saadeghi** [comment]
> dialogs (and popovers) are placed on the top layer and only other top layer

Yes. 

But to can put alerts inside the `<dialog>` tag, as a sibling of `modal-box` and it would be what you want.

Let me know if you have any questions.
